charts/consul/templates/server-config-configmap.yaml

```
@@ -27,7 +27,12 @@ data:
       "data_dir": "/consul/data",
       "domain": "{{ .Values.global.domain }}",
       "ports": {
+        {{- if not .Values.global.tls.enabled }}
         "grpc": 8502,
+        {{- end }}
+        {{- if .Values.global.tls.enabled }}
+        "grpc_tls": 8502,
+        {{- end }}
```

I found this the other day and thought I'd share, no need to change it! :D 
```suggestion
        {{ ternary "\"grpc_tls\":" "\"grpc\":" .Values.global.tls.enabled }} 8502,
```

---

oh interesting, thats a slick one! I'll skip that change for now since I personally find the existing format a little easier to read!